compiler/frontend/src/rcst.rs

```diff
@@ -17,7 +17,7 @@ impl From<CstKind<()>> for Cst<()> {
 
 impl ToRichIr for Rcst {
     fn build_rich_ir(&self, builder: &mut RichIrBuilder) {
-        builder.push(format!("{self:?}"), None, EnumSet::empty());
+        builder.push(format!("{self:#?}"), None, EnumSet::empty());
```

Comment:
The CST's rich IR is currently messed up and I suspect it comes from this change

[candy-lang/golden-irs@`a68dbb9b012ede17ab5d72dc6cc793aef4cf42e0_..3e506fd0a8da852079633eb18f932fe0baaeca27_`#diff-ff8b15b562](https://github.com/candy-lang/golden-irs/compare/a68dbb9b012ede17ab5d72dc6cc793aef4cf42e0_..3e506fd0a8da852079633eb18f932fe0baaeca27_#diff-ff8b15b5625a7cf5381cc45750d5c17c95d699e3eaf7f0d1de2cdd29cf7dfa6e)